broker/applications/osb-broker/src/api-controllers/ServiceBrokerApiController.js

```diff
@@ -140,7 +140,8 @@ class ServiceBrokerApiController extends FabrikBaseController {
         spec: params,
         status: {
           state: CONST.APISERVER.RESOURCE_STATE.IN_QUEUE
-        }
+        },
+        requestIdentity: _.get(req.headers, 'x-broker-api-request-identity')
```

Comment:
Read `x-broker-api-request-identity` from constants. (Please do this for other occurrences as well)

Reply:
Done[(Commit)](https://github.com/cloudfoundry-incubator/service-fabrik-broker/pull/1239/commits/86bfb2688e355784cb6595874498add0dc82c187)